src/Api/SecretManagerFeatures/Models/Request/SecretCreateRequestModel.cs

```diff
@@ -42,4 +42,3 @@ public IEnumerable<ValidationResult> Validate(ValidationContext validationContex
         }
     }
 }
-
```

Comment:
Were these new lines accidentally deleted or was this intentional?

Reply:
This was intentional - in some of the new secret files there are two new lines at the end, etc. I made it only one to be consistent with the other files in the repo.